src/openrct2/ride/VehicleRiderControl.cpp

```diff
@@ -10,7 +10,7 @@
 #include "../entity/EntityRegistry.h"
 #include "Vehicle.h"
 
-constexpr int operator"" _MPH(unsigned long long x)
+constexpr int operator"" _MPH(unsigned long long x) noexcept
```

Comment:
Could you explain this change?

Reply:
This is something I forgot when refactoring the `_MPH` operator. AFAIK it is good practice to label function as `noexcept` when this is the case, therefore added.